src/kernels/MIOpenLayerNorm.cpp

```diff
@@ -109,16 +102,15 @@ extern "C" __global__ void LayernormFwdContiguous(const FLOAT* __restrict__ x,
     // forward calculation
     for(uint64_t i = lid; i < inner_size; i += LOCAL_SIZE)
     {
-        uint64_t idx = gid * inner_size + i;
+        size_t idx = gid * inner_size + i;
 
         FLOAT_ACCUM pweight;
         FLOAT_ACCUM pbias;
 
-        pweight = mode ? CVT_FLOAT2ACCUM(1) : CVT_FLOAT2ACCUM(weight[i]);
-        pbias   = mode ? CVT_FLOAT2ACCUM(0) : CVT_FLOAT2ACCUM(bias[i]);
+        pweight = mode ? CVT_FLOAT2ACCUM(weight[i]) : CVT_FP32_2ACCUM(1.0f);
+        pbias   = mode ? CVT_FLOAT2ACCUM(bias[i]) : static_cast<FLOAT>(0);
```

Comment:
Change of behavior. Bugfix?

Reply:
Yes, It was working incorrectly previously.

Reply:
@seungmanhan Thanks [resolved]